src/include/sof/audio/format_hifi3.h

```diff
@@ -135,14 +135,26 @@ static inline ae_int16x4 vec_sat_int16x4(int32_t x, int32_t y, int32_t z, int32_
 	/* Saturate all values to 16-bit and pack them */
 	return AE_SAT16X4(d0, d1);
 }
+
```

Comment:
No issues in the code but the commit text indicates an issue in the code that I don't think exists. I do agree with perf improve with this AE code version. So better to talk in commit text about optimize and not something that sounds like a bug fix.